### PR TITLE
[fix] Do not write 'Location' with geolocation (0.0,0.0)

### PR DESCRIPTION
**Pull request by woshilapin**

### CODE REVIEW

File: src/netex_france/offer.rs [hunk 1]
```diff
@@ -448,15 +452,18 @@ impl<'a> OfferExporter<'a> {
         Exporter::generate_id(&order_id, object_type)
     }
 
-    fn generate_location(&self, coord: &'a Coord) -> Result<Element> {
+    fn generate_location(&self, coord: &'a Coord) -> Result<Option<Element>> {
+        if coord.lat == 0.0 && coord.lon == 0.0 {
```
Comment (ArnaudOggy) [inline src/netex_france/offer.rs]:
```suggestion
       if *coord == Coord::default() {
```

File: src/netex_france/stops.rs [hunk 2]
```diff
@@ -283,7 +297,10 @@ impl<'a> StopExporter<'a> {
         })
     }
 
-    fn generate_centroid(&self, coord: &'a Coord) -> Result<Element> {
+    fn generate_centroid(&self, coord: &'a Coord) -> Result<Option<Element>> {
+        if coord.lat == 0.0 && coord.lon == 0.0 {
```
Comment (ArnaudOggy) [inline src/netex_france/stops.rs]:
```suggestion
       if *coord == Coord::default() {
```

File: src/netex_france/offer.rs (hunk 1 above)
Comment (woshilapin) [reply src/netex_france/offer.rs]: Done in 86a310621db97e5c506bc77a9c545e4559a867f9.

File: src/netex_france/stops.rs (hunk 2 above)
Comment (woshilapin) [reply src/netex_france/stops.rs]: Done in 86a310621db97e5c506bc77a9c545e4559a867f9.

**Comment by woshilapin** [comment]
I also improved documentation here b26ea6bb2ba198df0a52d47436431dfd1bc5ea95.

**Pull request merged**